src/init/starship.bash

```diff
@@ -111,9 +113,9 @@ else
 
     # Finally, prepare the precmd function and set up the start time. We will avoid to
     # add multiple instances of the starship function and keep other user functions if any.
-    if [[ -z "$PROMPT_COMMAND" ]]; then
+    if [[ -n "${PROMPT_COMMAND-}" ]]; then
         PROMPT_COMMAND="starship_precmd"
-    elif [[ "$PROMPT_COMMAND" != *"starship_precmd"* ]]; then
+    elif [[ -n "${PROMPT_COMMAND-}" && "$PROMPT_COMMAND" != *"starship_precmd"* ]]; then
```

Comment:
I would prefer this or keeping it as it was since this is already checked above.
```suggestion
    elif [[ "${PROMPT_COMMAND-}" != *"starship_precmd"* ]]; then
```

Reply:
@davidkna I kept it as it is because you are right again, it is checked already.